SP-1606: Updates to describe the content of DP1 and updates to the AP plans (#41)
lsst/rtn-011: intro.tex

\section{Rubin Early Science Program}

Community expectations for early science with Rubin are high due to the transformative nature of the LSST data and the densely-sampled observations planned during the commissioning period.

> Rubin Observatory's \emph{Early Science Program} was conceived to provide Rubin data rights holders with early access to the data products and services necessary to produce high-impact  science during time between commissioning through, and including, the first data release, Data Release 1 (DR1). 

Author: fixed

The Early Science program is motivated by the desire to:
\begin{itemize}

> \item enable high-impact science with LSST as early as possible;

Reviewer: Should this be "with Rubin Observatory data"? Asking because DP1 and DP2 are Rubin data before the start of the 10-year LSST.

Author: Both actually as DR1 is "Early Science" -- good catch

All Early Science data products are opportunistic on the commissioning activities, meaning that a detailed description will only be possible once the commissioning data have been acquired and analyzed. 
Consequently, while the Operations team will do its best to deliver the maximum of early data, data product types and services, any statement on the contents of the Data Previews, early Alert Stream and supporting services is subject to change up until the release date. 

\subsection{Factors Impacting the Early Science Program }
\label{ssec:impact}

Factors affecting the schedule and contents of the Early Science program can be broadly grouped into technical considerations and policy considerations. 

Technical considerations include:
\begin{itemize}
\item The operational status of the observatory and progress of system integration and test activities in commisioning; 
\item The nature and quality of the data collected during commissioning; 
\item The readiness of the data processing pipelines and data distribution and access services. 
\end{itemize}
Policy factors include:
\begin{itemize}
\item There is a 30-day embargo on all pixel data during commissioning;

> \item There is an 80-hour embargo on all pixel data throughout the entirety of Pperations, including the SV surveys and the LSST; 

Reviewer 2: ```suggestion
\item There will be an 80-hour embargo on all pixel data throughout the LSST; 
```

> \item The construction security review, which must be successfully completed prior to the release of any Prompt data products; 

Reviewer: Suggest to make this a complete sentence, for consistency with the other bullets.

"A construction security review must be successfully completed prior to the release of any Prompt data products;"

> \item The Rubin First Look (RFL) media event, \citeds{rtn-083}. Currently scheduled for  June 2025. No LSST data may be released before this date. 

Reviewer 2: ```suggestion
\item The Rubin First Look (RFL) media event, \citeds{rtn-083}. Currently scheduled for  June 2025. No Rubin image data may be released before this date. 
```

Reviewer: For clarity, perhaps we want to make clear that this is a milestone rather than a fixed date

"before this date" -> "before this event" or "before this milestone"

The Early Science program is motivated by the desire to:
\begin{itemize}
\item enable high-impact science with LSST as early as possible;

> \item provide early access to both static-sky and time-domain science-ready data products to support the community to prepare in advance of the first LSST data release;

Author: The first Data Release is DR1. DP0/1/2 are Data Previews, not Data Releases. But to be clear, I added (DR1)

> \item provide early access to both static-sky and time-domain science-ready data products to support the community to prepare in advance of the first LSST data release;

Reviewer 3: My main concern was about lowercase "data release" meaning something more generic than a "Data Release".  (We have the same ambiguity in SPHEREx.)  It's the capitalization that's the most important here.

> \item provide early access to both static-sky and time-domain science-ready data products to support the community to prepare in advance of the first LSST data release;

Author: yes, I also changed that

\item enable early time-domain astronomy via Alert Production; and 
\item help drive the development of Rubin operations capabilities prior to survey start and prepare the team to be operations-ready.
\end{itemize}

\subsection{Elements of the Early Science Program}

The Early Science Program consists of the following elements:
\begin{itemize}
	\item A series of three \textbf{Data Previews (DP)}, DP0, DP1 and DP2,  based on either simulated LSST-like data or data taken during the Rubin Observatory commissioning period. 
	\item A world-public \textbf{stream of Alerts} from transient, variable, and moving sources that will be scaled up continuously during commissioning and the first year of the survey. 
	\item  \textbf{Template generation}, both prior to the start of regular survey operations based on data collected during the commissioning period with LSSTCam, and incrementally during the first year of regular survey operations  to maximize the number of templates available for Alert Production in year 1. 

> 	\item \textbf{LSST Data Release 1 (DR1)}, which will be based on the Data Release Processing (DRP) of the first six months of LSST data.

Author: I don't consider commissioning data to be LSST data  - it is Rubin data, but LSST data is only that data that follows the LSST observing strategy

> 	\item \textbf{LSST Data Release 1 (DR1)}, which will be based on the Data Release Processing (DRP) of the first six months of LSST data.

Reviewer 3: I understood that's what you _intended_ but I think the reader might not automatically apply that nuance; adding "survey" to the text just fends off a possible casual misunderstanding.

> 	\item \textbf{LSST Data Release 1 (DR1)}, which will be based on the Data Release Processing (DRP) of the first six months of LSST data.

Author: Then it becomes "Legacy Survey of Space and Time Survey data" -- which I try to avoid

> 	\item \textbf{LSST Data Release 1 (DR1)}, which will be based on the Data Release Processing (DRP) of the first six months of LSST data.

Author: How about "of the first six months of LSST data following the baseline survey strategy" ?

> 	\item \textbf{LSST Data Release 1 (DR1)}, which will be based on the Data Release Processing (DRP) of the first six months of LSST data.

Reviewer 3: Yes, that's fine.  (I also suggested on Slack the shorter "first six months of LSST (survey) data." to avoid the repetition you mentioned.)

\label{ssec:impact}

Factors affecting the schedule and contents of the Early Science program can be broadly grouped into technical considerations and policy considerations. 

Technical considerations include:
\begin{itemize}
\item the operational status of the observatory and progress of system integration and test activities; 
\item the nature and quality of the data collected during commissioning; 
\item the readiness of the data processing pipelines and data distribution and access services. 
\end{itemize}
Policy factors include:
\begin{itemize}
\item the 30-day embargo on all pixel data during commissioning;
\item the 80-hour embargo on all pixel data throughout operations, including the SV surveys and the LSST; 
\item the security review, which must be successfully completed prior to the release of any prompt data; 
\item The First Look media event, before which, no LSST data may be released.
\end{itemize}
 
In this document, the term ``stretch goal'' will be used in cases where any uncertainty is due to a technical consideration and ``TBD'' (To Be Decided) will be used when the influencing factor is of a policy nature. 

> %%%% ---- V5

Author: I will clean all old commented text out before issuing
